rust/ares/src/jets/math.rs

```diff
@@ -543,6 +543,12 @@ pub fn jet_cut(stack: &mut NockStack, subject: Noun) -> Result<Noun, JetErr> {
     let run = raw_slot(arg, 13).as_direct()?.data() as usize;
     let atom = raw_slot(arg, 7).as_atom()?;
 
+    if run == 0 {
+        unsafe {
+            return Ok(DirectAtom::new_unchecked(0).as_noun());
+        }
```

Comment:
Typically we use the `D` shortcut for this.

```suggestion
        return Ok(D(0));
```